zeppelin-web/src/app/notebook/paragraph/paragraph-chart-selector.html

```diff
@@ -13,49 +13,55 @@
 -->
 
 <div id="{{paragraph.id}}_switch"
-     ng-if="(paragraph.result.type == 'TABLE' || apps.length > 0 || suggestion.available && suggestion.available.length > 0) && !asIframe && !viewOnly"
+     ng-if="(paragraph.result.type == 'TABLE' || paragraph.result.type == 'NETWORK' || apps.length > 0 || suggestion.available && suggestion.available.length > 0) && !asIframe && !viewOnly"
      class="btn-group"
      style="margin-bottom: 10px;">
 
   <button type="button" class="btn btn-default btn-sm"
-          ng-if="paragraph.result.type == 'TABLE'"
+          ng-if="paragraph.result.type == 'TABLE' || paragraph.result.type == 'NETWORK'"
```

Comment:
I think all other types of visualizations - table, bar, line, area, scatter shouldn't be selectable when result type is 'NETWORK' while they can not display it.